docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Grafana Mimir version 2.2 release notes"
+menuTitle: "V2.2 release notes"
+description: "Release notes for Grafana Mimir version 2.2"
+weight: 300
+---
+
+# Grafana Mimir version 2.2 release notes
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Highlights include the top features, enhancements, and bugfixes in this release. If you are upgrading from Grafana Mimir 2.1, there is migration-related information as well.
+For the complete list of changes, see the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
```

Comment:
It's such a low-level change, that I have difficulty phrasing it. What about:

> Store-gateway can now optionally prepopulate the file system cache when memory-mapping index-header files. This can help store-gateway to avoid looking stuck while processing index-headers. Feature can be enabled with new experimental flag `-blocks-storage.bucket-store.index-header.map-populate-enabled`.

Reply:
LGTM, I would just do the following change:

> while processing index-headers
while loading index-headers

> It's such a low-level change
True it's low level, but impactful.

Reply:
> > It's such a low-level change
> True it's low level, but impactful.

I didn't mean to downplay it, I was just wondering how to explain it :)